### ISSUE TITLE
Owner of Find command window is lost on ScriptEditor.hide()

### ISSUE THREAD

**Issue by melvingelbard** [qupath/qupath#893]
## Bug report

**Describe the bug**
When the script editor is closed after having used the 'Find command' at least once, the owner of the 'Find command' is lost. This means that it will 'fall behind' the main QuPath window if the focus changes.

**To Reproduce**
1. Open the script editor
2. Open the 'Find command' (Ctrl/CMD + F)
3. Notice how the 'Find command' window stays on top of the main QuPath viewer window, even when clicking on the viewer
4. Close the 'Find command' window
5. Close the script editor window
6. Reopen the script editor window
7. Reopen the 'Find command' window (Ctrl/CMD + F)
8. Notice how the 'Find command' window is lost behind the main QuPath viewer when clicking on the viewer

**Expected behavior**
Consistent behaviour where the 'Find command' window always stays on top.

**Desktop (please complete the following information):**
 - OS: All
 - QuPath Version: 0.3.2

**Additional context**
Basically, the 'Find command' window is not recreated at every call. It is initialised once, then reused.
When it is initialised, the owner (i.e. Script editor's dialog) is set. However, when the script editor's dialog is closed, the 'Find command' window's ownership does not exist anymore and is never re-set again (since the same 'Find command' dialog is reused).

**Comment by petebankhead** [comment]
Fixed in https://github.com/qupath/qupath/pull/898 (I think, please reopen if not)
